cli_reference/basic_cli_operations.adoc

```diff
@@ -172,13 +172,16 @@ OpenShift objects at the project level.
 
 The simplest way to create a new project is:
 
-****
-`$ osadm new-project _<project_name>_ --display-name=_<display_name>_ --description=_<description>_ --admin=_<admin_username>_ --node-selector=_<node_label_selector>_`
-****
+----
+$ osadm new-project <project_name> --display-name=<display_name>
+--description=<description> --admin=<admin_username>
+--node-selector=<node_label_selector>
+----
 
 The following example creates a new project called `test` that appears in the
-Management Console as "Openshift 3 Sample", with `test-admin` as the project
-admin and launches any pods onto nodes matching label `environment : test`.
+Management Console as "OpenShift 3 Sample" with `test-admin` as the project
+administrator. Any pods are launched onto nodes with the matching label
```

Comment:
Passive voice.

Could you write this instead?

"Pods launch onto nodes with the matching label...."

What's the purpose of saying "any"?